spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/SvnKitEnvironmentRepository.java

```diff
@@ -183,9 +183,9 @@ private File getSvnPath(File workingDirectory, String label) {
 		File svnPath = new File(workingDirectory, label);
 		if(!svnPath.exists()) {
 			svnPath = new File(workingDirectory, "branches" + File.separator + label);
-		}
-		if(!svnPath.exists()) {
-			svnPath = new File(workingDirectory, "tags" + File.separator + label);
+			if(!svnPath.exists()) {

```

Comment:
This is covered in IntegrationTest - public void invalidLabel(), if the folder doesn't exists an exception will be thrown.  Either way you will be returning a path that does not exists (whether it is the tags folder or trunk folder should be irrevelant)